src/internal/hooks/use-virtual/index.ts

```diff
@@ -74,9 +76,15 @@ export function useVirtual<Item extends object>({
     [items, rowVirtualizer.virtualItems]
   );
 
+  // Adjust totalSize to account for 1px overlap per item (matching the position adjustment in renderOptions: /select/utils/render-options.tsx)
```

Comment:
I would remove the "(matching the position adjustment in renderOptions: /select/utils/render-options.tsx)", which is specific to select and multiselect, while the use-virtual is an abstract utility and is also used in autosuggest.